app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -33,23 +29,13 @@ private val HIDE_AFTER_MILLIS = TimeUnit.SECONDS.toMillis(3)
 class CursorView(context: Context, attrs: AttributeSet) : View(context, attrs) {
 
     private val hideHandler = CursorHideHandler(this)
-
-    // The radius is half the size of the container its drawn in.
-    private val radius = context.resources.getDimensionPixelSize(R.dimen.remote_cursor_size) / 2f
-    private val paint = Paint().apply {
-        style = Paint.Style.FILL
-        alpha = CURSOR_ALPHA
-        isAntiAlias = true
-        shader = RadialGradient(radius, radius, /* radius */ radius,
-                ContextCompat.getColor(context, R.color.teal50), ContextCompat.getColor(context, R.color.photonBlue50),
-                Shader.TileMode.CLAMP)
-    }
+    private val CROSSHAIR_OFFSET = 6
 
     @UiThread
     fun updatePosition(x: Float, y: Float) {
-        // In onDraw, we offset the initial position from the origin: we undo that offset here.
-        translationX = x - radius
-        translationY = y - radius
+        // Offset the position to better align with the crosshairs.
+        translationX = x - CROSSHAIR_OFFSET
```

Comment:
I feel like there is probably a better way to do this. But when I tried to move the cursor it always got clipped by the view

Reply:
The position coming from `CursorViewModel` has no knowledge of the view hierarchy (beyond the window bounds) so it's position will be the center of the view while `translation` affects the top/left bounds.

I think you'll actually want to subtract `height / 2` and `width / 2` from translation (which was `radius` in the existing implementation).

Reply:
This is true, but the click point is no longer the center, its only offset by 7 points.  Because the cursor can be positioned by a fraction of a point it looks like the click can get rounded when going into the webview. I spent some time trying to fix but couldn't find the best spot to put it in.

![img_1196](https://user-images.githubusercontent.com/1250545/36706649-b023eb62-1b1f-11e8-885f-fbc600b26200.jpg)

Reply:
tbh, this is probably good enough - no one is going to be this accurate when using the remote on their TV.

That being said, if we want to do better, we can put it in a place that's "good enough" and fat finger it, letting WebView handle disambiguation: we're sending MotionEvents, which are usually triggered by touch events, and you can actually specify the size of the press with this obtain method: https://developer.android.com/reference/android/view/MotionEvent.html#obtain(long,%20long,%20int,%20float,%20float,%20float,%20float,%20int,%20float,%20float,%20int,%20int)